shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Question.kt

```diff
@@ -1,29 +1,63 @@
 package co.nimblehq.ic.kmm.suv.domain.model
 
+const val QUESTION_DISPLAY_TYPE_INTRO = "intro"
+const val QUESTION_DISPLAY_TYPE_STAR = "star"
+const val QUESTION_DISPLAY_TYPE_HEART = "heart"
+const val QUESTION_DISPLAY_TYPE_SMILEY = "smiley"
+const val QUESTION_DISPLAY_TYPE_CHOICE = "choice"
+const val QUESTION_DISPLAY_TYPE_NPS = "nps"
+const val QUESTION_DISPLAY_TYPE_TEXTAREA = "textarea"
+const val QUESTION_DISPLAY_TYPE_TEXTFIELD = "textfield"
+const val QUESTION_DISPLAY_TYPE_DROPDOWN = "dropdown"
+const val QUESTION_DISPLAY_TYPE_OUTRO = "outro"
+const val QUESTION_DISPLAY_TYPE_UNSUPPORTED = "unsupported"
+
 data class Question(
     val id: String,
     val text: String,
     val displayOrder: Int,
-    val displayType: QuestionDisplayType,
+    val displayType: String,
     val pick: String,
     val coverImageUrl: String,
     val answers: List<Answer>
 )
 
-enum class QuestionDisplayType(val value: String) {
-    INTRO("intro"),
-    STAR("star"),
-    HEART("heart"),
-    SMILEY("smiley"),
-    CHOICE("choice"),
-    NPS("nps"),
-    TEXTAREA("textarea"),
-    TEXTFIELD("textfield"),
-    DROPDOWN("dropdown"),
-    OUTRO("outro"),
-    UNSUPPORTED("unsupported"),
+sealed class QuestionDisplayType(val value: String) {
+
+    object Intro : QuestionDisplayType(QUESTION_DISPLAY_TYPE_INTRO)
+    object Star : QuestionDisplayType(QUESTION_DISPLAY_TYPE_STAR)
+    object Heart : QuestionDisplayType(QUESTION_DISPLAY_TYPE_HEART)
+    object Smiley : QuestionDisplayType(QUESTION_DISPLAY_TYPE_SMILEY)
+    data class Choice(val answers: List<String> = emptyList()) :
+        QuestionDisplayType(QUESTION_DISPLAY_TYPE_CHOICE)
```

Comment:
I think we don't need default value here

```suggestion
    data class Choice(val answers: List<String>) :
        QuestionDisplayType(QUESTION_DISPLAY_TYPE_CHOICE)
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/118/commits/9596bc9bf072bfdede796aea64e6e6536a022a88